components/Card/card.css

```diff
@@ -0,0 +1,78 @@
+@import "/docs.css";
+@import "components\Button\button.css";
+/* @import "/components/Utilities/utilities.css"; */
+
+.card{
+    display: flex;
+    flex-direction: column;
+    max-width:min-content;
+    border-radius: 10px;
```

Comment:
Next to card class you have already used br-md in line 74 in card.html. So you can remove this here because both refers to the same right ?

Reply:
Yes, right. It will be removed as soon as i add utility classes.